### PR TITLE
add exchange section

### PR DESCRIPTION
**Pull request by jaxonL**
### changes
- add `renderGeographicMap` which can render a map with geoJson data
- add exchange section
- add styling class: a "small" class on `span` elements makes it so that the text is... small
- add a world.geo.json map that can be used as the base for other data update
- add a migrate file for exchange data to create the exchange geo json file

### other
- update the readme with missing info
- update some text/calculations of the relationships section
  - had additional data row
- add .json typing so that we can directly import json data

### CODE REVIEW

File: content/exchange.html [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+<div class="vb"></div>
+
+<h3>SE Abroad</h3>
+<p class="full">
+    <span class="big">
+        When asked whether they would suggest going on exchange to future SEs, 7
+        out of 8 said yes.
+    </span>
+    SE students who have gone on exchange usually went on a study term after their
+    2A one. A good amount went on exchange in their 3B term. All but one respondent
```
Comment (roxelo) [inline content/exchange.html]: 2A term

File: content/exchange.html [hunk 2]
```diff
@@ -0,0 +1,141 @@
...
+    not match the typical UW start/end dates.
+    <br /><br />
+    Connecting with the locals and making friends in a meaningful capacity. Some
+    exchange schools have a more mature program than others where it is easier to
+    make friends. If I were to do exchange again I would opt for a school where a
+    lot of international students go and where the program is well established.
+    <br /><br />
+    Balancing schoolwork/wanting to graduate at the same time as the SE cohort with
+    taking advantage of being in a new country. Oh, and the timezone differences when
+    communicating with friends, family, group project members and NA interviewers
```
Comment (roxelo) [inline content/exchange.html]: NA?

**Review by roxelo — COMMENTED** [review]
A few nits otherwise looks good to me

File: content/exchange.html (hunk 2 above)
Comment (jaxonL) [reply content/exchange.html]: probably north american. i will change that! 👀

**Pull request merged**